Neos.ContentRepository.Security/Classes/Authorization/Privilege/Node/NodePrivilegeContext.php

```diff
@@ -61,7 +61,7 @@ public function isAncestorNodeOf(string $nodePathOrIdentifier): bool
             return $nodePathOrResult;
         }
 
-        return str_starts_with($nodePathOrResult, $this->getSubgraph()->retrieveNodePath($this->node->nodeAggregateId)->value);
+        return str_starts_with($nodePathOrResult, $this->getSubgraph()->retrieveNodePath($this->node->nodeAggregateId)->serializeToString());
```

Comment:
This would require `$nodePathOrIdentifier` to have a syntax of `<NodeType>/foo/bar` – is that really the case? If so, the doc comments should be adjusted accordingly